.babelrc

```diff
@@ -1,23 +1,35 @@
 {
+    "sourceType": "script",
+    "generatorOpts": {
+        "retainLines": true
+    },
     "env": {
         "development": {
             "presets": [
-                "es2015-without-strict"
+                [
+                  "@babel/preset-env",
+                  {
+                  }
+                ]
             ],
             "plugins": [
-                ["transform-runtime", {
-                    "polyfill": false
+                ["@babel/plugin-transform-runtime", {
+                    "modules": false
                 }]
             ]
         },
         "test": {
             "presets": [
-                "es2015-without-strict"
+                [
+                  "@babel/preset-env",
+                  {
+                  }
+                ]
```

Comment:
You can just write "@babel/preset-env", no need for the options syntax.

Reply:
Yeah, there has been a lot of changes back and forth. I'll clean it up when it's ready to merge.